src/test/java/com/google/sps/EventTest.java

```diff
@@ -35,90 +39,58 @@
   // files for event 1
   private static final String GOLDEN_GATE_PARK = "GGPark";
   private static final String ADDRESS =  "4265 24th Street San Francisco, CA, 94114";
-  
 
-  @Test
-  public void testMinuteToHHMMFormatOneHour() {
-    String output = Event.convertToFormat(HOUR);
-    Assert.assertEquals("0100", output);
-  }
-
-  @Test
-  public void testMinuteToHHMMFormatHalfHour() {
-    String output = Event.convertToFormat(HALF_HOUR);
-    Assert.assertEquals("0030", output);
-  }
-
-  @Test
-  public void testMinuteToHHMMFormatNinetyMin() {
-    String output = Event.convertToFormat(NINETY_MIN);
-    Assert.assertEquals("0130", output);
-  }
-
-  @Test
-  public void testMinuteToHHMMFormatHourFivetyFive() {
-    String output = Event.convertToFormat(HOUR_FIFTY_FIVE);
-    Assert.assertEquals("0155", output);
-  }
-
-  @Test
-  public void testMinuteToHHMMFormatThreeHour() {
-    String output = Event.convertToFormat(THREE_HOURS);
-    Assert.assertEquals("0300", output);
-  }
+  private static final LocalDateTime DEF_LDT = LocalDateTime.of(LocalDate.parse("2020-06-25"), LocalTime.of(10, 0));
 
   @Test(expected = IllegalArgumentException.class)
-  public void testMinuteToHHMMFormatLessThanZero() {
-    Event.convertToFormat(-1);
+  public void testcheckTravelTimeMinPossibleTime() {
```

Comment:
nit: I'd rename this test to make it clearer that you expect a failure, e.g. testTravelTimeBelowMinPossible